sonic-net/sonic-buildimage
# [201911][sonic-utilties] Submodule Update

#### Why I did it
To pick up - 

**ef2a0cd0**      [201911] [multi_asic] Script to monitor errors on internal links  ([#2971](https://github.com/sonic-net/sonic-utilities/pull/2971))
**1252e31b**     Changes to separate UT data for internal link monitor ([#2976](https://github.com/sonic-net/sonic-utilities/pull/2976))
**3e6654e**   [[201911] [multi-asic] Unit test fix for internal link monitoring ([#2977](https://github.com/sonic-net/sonic-utilities/pull/2977))

##### Work item tracking
- Microsoft ADO **(number only)**: 15439060

#### How I did it
Updated sonic-utilities submodule

#### How to verify it
Ensure the script is available and the build with new unit test cases is successful



#### Which release branch to backport (provide reason below if selected)




#### Tested branch (Please provide the tested image version)




#### Description for the changelog




#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
